.github/workflows/linters.yml

```diff
@@ -0,0 +1,19 @@
+name: Linters
+
+on: pull_request
```

Comment:
In the pull request summary of Frontend repository, you should have a link to your Backend pull request :+1: to follow the requirements :blush: